Docs/Configuration.tex

```diff
@@ -1343,15 +1343,14 @@ \subsection{Patch Properties}\label{booterpropspatch}
   \texttt{Find}\\
   \textbf{Type}: \texttt{plist\ data}\\
   \textbf{Failsafe}: Empty\\
-  \textbf{Description}: Data to find. This must equal to \texttt{Replace} in size.
+  \textbf{Description}: Data to find. Must be equal to \texttt{Replace} in size.
 
 \item
   \texttt{Identifier}\\
   \textbf{Type}: \texttt{plist\ string}\\
-  \textbf{Failsafe}: Empty\\
+  \textbf{Failsafe}: Empty (Match any booter)\\
```

Comment:
This is an overlook actually. It should be `Any` for failsafe. I fixed the code in master. Specifying an empty value is not allowed here.

Reply:
OK. What should it say?

Reply:
Just to be clear, do you mean:
`\textbf{Failsafe}:  \texttt{Any} (Match any booter)\\`

Reply:
Yeah, thanks